code/modules/surgery/organs/augments_arms.dm

```diff
@@ -233,3 +233,13 @@
 	name = "surgical toolset implant"
 	desc = "A set of surgical tools hidden behind a concealed panel on the user's arm."
 	contents = newlist(/obj/item/retractor/augment, /obj/item/hemostat/augment, /obj/item/cautery/augment, /obj/item/surgicaldrill/augment, /obj/item/scalpel/augment, /obj/item/circular_saw/augment, /obj/item/surgical_drapes)
+
+/obj/item/organ/cyberimp/arm/syndie_mantis
+	name = "G.O.R.L.E.X. mantis blade implants"
+	desc = "Modernized mantis blade designed coined by Tiger operatives, much sharper blade with energy actuators makes it a much deadlier weapon."
```

Comment:
More grammar fix.

"Modernized mantis blade designed and coined by Tiger operatives. Energy actuators makes the blade a much deadlier weapon."